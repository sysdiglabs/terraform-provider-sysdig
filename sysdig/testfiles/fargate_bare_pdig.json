[
    {
        "Name": "defaultInstrumentation",
        "Image": "test_image:latest",
        "EntryPoint": [
            "/bin/test"
        ],
        "Environment": [
            {
                "Name": "GIMME",
                "Value": "friedChicken"
            },
            {
                "Name": "THE_ANSWER_IS",
                "Value": "42"
            }
        ]
    },
    {
        "Name": "barePdig",
        "Image": "test_image:latest",
        "EntryPoint": [
            "/bin/test"
        ],
        "Environment": [
            {
                "Name": "GIMME",
                "Value": "friedChicken"
            },
            {
                "Name": "THE_ANSWER_IS",
                "Value": "42"
            }
        ]
    },
    {
        "Name": "skipped",
        "Image": "test_image:latest",
        "EntryPoint": [
            "/bin/test"
        ],
        "Environment": [
            {
                "Name": "GIMME",
                "Value": "friedChicken"
            },
            {
                "Name": "THE_ANSWER_IS",
                "Value": "42"
            }
        ]
    }
]