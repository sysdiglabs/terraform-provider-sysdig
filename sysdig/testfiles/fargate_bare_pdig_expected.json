[
    {
        "Name": "defaultInstrumentation",
        "Image": "test_image:latest",
        "EntryPoint": [
            "/opt/draios/bin/instrument"
        ],
        "Command": [
            "/bin/test"
        ],
        "Environment": [
            {
                "Name": "SYSDIG_ORCHESTRATOR_PORT",
                "Value": "orchestrator_port"
            },
            {
                "Name": "SYSDIG_COLLECTOR",
                "Value": "collector_host"
            },
            {
                "Name": "SYSDIG_COLLECTOR_PORT",
                "Value": "collector_port"
            },
            {
                "Name": "SYSDIG_ACCESS_KEY",
                "Value": "sysdig_access_key"
            },
            {
                "Name": "SYSDIG_LOGGING",
                "Value": "sysdig_logging"
            },
            {
                "Name": "SYSDIG_PRIORITY",
                "Value": "priority"
            },
            {
                "Name": "SYSDIG_SIDECAR",
                "Value": ""
            },
            {
                "Name": "SYSDIG_ORCHESTRATOR",
                "Value": "orchestrator_host"
            },
            {
                "Name": "GIMME",
                "Value": "friedChicken"
            },
            {
                "Name": "THE_ANSWER_IS",
                "Value": "42"
            }
        ],
        "LinuxParameters": {
            "Capabilities": {
                "Add": [
                    "SYS_PTRACE"
                ]
            }
        },
        "VolumesFrom": [
            {
                "ReadOnly": true,
                "SourceContainer": "SysdigInstrumentation"
            }
        ]
    },
    {
        "Name": "barePdig",
        "Image": "test_image:latest",
        "EntryPoint": [
            "/opt/draios/bin/instrument"
        ],
        "Command": [
            "/bin/test"
        ],
        "Environment": [
            {
                "Name": "SYSDIG_ORCHESTRATOR_PORT",
                "Value": "orchestrator_port"
            },
            {
                "Name": "SYSDIG_COLLECTOR",
                "Value": "collector_host"
            },
            {
                "Name": "SYSDIG_COLLECTOR_PORT",
                "Value": "collector_port"
            },
            {
                "Name": "SYSDIG_ACCESS_KEY",
                "Value": "sysdig_access_key"
            },
            {
                "Name": "SYSDIG_LOGGING",
                "Value": "sysdig_logging"
            },
            {
                "Name": "SYSDIG_PRIORITY",
                "Value": "priority"
            },
            {
                "Name": "SYSDIG_SIDECAR",
                "Value": ""
            },
            {
                "Name": "SYSDIG_ORCHESTRATOR",
                "Value": "orchestrator_host"
            },
            {
                "Name": "GIMME",
                "Value": "friedChicken"
            },
            {
                "Name": "THE_ANSWER_IS",
                "Value": "42"
            },
            {
                "Name": "__INSTRUMENTATION_WRAPPER",
                "Value": "/opt/draios/bin/pdig,-C,-t,-1"
            }
        ],
        "LinuxParameters": {
            "Capabilities": {
                "Add": [
                    "SYS_PTRACE"
                ]
            }
        },
        "VolumesFrom": [
            {
                "ReadOnly": true,
                "SourceContainer": "SysdigInstrumentation"
            }
        ]
    },
    {
        "Name": "skipped",
        "Image": "test_image:latest",
        "EntryPoint": [
            "/bin/test"
        ],
        "Environment": [
            {
                "Name": "GIMME",
                "Value": "friedChicken"
            },
            {
                "Name": "THE_ANSWER_IS",
                "Value": "42"
            }
        ]
    },
    {
        "EntryPoint": [
            "/opt/draios/bin/logwriter"
        ],
        "Image": "workload_agent_image",
        "Name": "SysdigInstrumentation",
        "RepositoryCredentials": {
            "CredentialsParameter": "image_auth_secret"
        },
        "Environment": [
            {
                "Name": "SYSDIG_ORCHESTRATOR_PORT",
                "Value": "orchestrator_port"
            },
            {
                "Name": "SYSDIG_COLLECTOR",
                "Value": "collector_host"
            },
            {
                "Name": "SYSDIG_COLLECTOR_PORT",
                "Value": "collector_port"
            },
            {
                "Name": "SYSDIG_ACCESS_KEY",
                "Value": "sysdig_access_key"
            },
            {
                "Name": "SYSDIG_LOGGING",
                "Value": "sysdig_logging"
            },
            {
                "Name": "SYSDIG_PRIORITY",
                "Value": "priority"
            },
            {
                "Name": "SYSDIG_SIDECAR",
                "Value": ""
            },
            {
                "Name": "SYSDIG_ORCHESTRATOR",
                "Value": "orchestrator_host"
            },
            {
                "Name": "GIMME",
                "Value": "friedChicken"
            },
            {
                "Name": "THE_ANSWER_IS",
                "Value": "42"
            }
        ]
    }
]